.github/workflows/staging-sgx-restart.yml

```diff
@@ -0,0 +1,28 @@
+name: Staging SGX parachain
+
+on:
+  workflow_dispatch:
+    inputs:
+      action:
+        type: choice
+        description: Select the action
+        required: true
+        default: stop
+        options: 
+        - stop
+        - start
+
+jobs:
+  action:
+    runs-on: self-hosted #tee-staging (162.19.84.27)
+
+    steps:
+      - name: restarting the service
+        if: "${{ github.event.inputs.action == 'stop' }}"
+        run: |
+          sudo systemctl stop integritee.service
+
+      - name: stoping the service
+        if: "${{ github.event.inputs.action == 'start' }}"
+        run: |
+          sudo systemctl start integritee.service
```

Comment:
Maybe we should also collect the logs whenever we do a full reset (restart)

Reply:
added restart

not sure what you mean by `Maybe we should also collect the logs whenever we do a full reset (restart)`

Reply:
will the logs be overwritten? Maybe we want to keep the previous logs before restarting :)

Reply:
I tested the restart by adding stop and start actions respectively. 
But in some cases, as I faced just now while running the [workflow] (https://github.com/litentry/litentry-parachain/actions/runs/4342133882/jobs/7582587026), if the service is not running on the host, the workflow will fail.
Adding stop, start and restart all three options in case, the service is not already running to stop.

Reply:
for the logs, maybe it's easy to handle inside python script when do the 'graceful_exit'

Reply:
> for the logs, maybe it's easy to handle inside python script when do the 'graceful_exit'

Yes I mean we need to upload this logs somewhere, as this one is triggered by GHA. 

Whoever ran this GHA doesn't necessarily have the permission to log on to the host to check the logs (e.g. IDH colleagues)